### DISCUSSION TITLE
Evaluation procedure update!

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas22_competition#43] [Announcements]
Dear teams, 

we have performed the first run of the evaluations. Our automated system still needs a few tweaks, but we will be working on getting it up and running. In the meantime, we will do evaluations manually.

Alongside the [existing instructions](https://github.com/larics/icuas22_competition/discussions/39), here are a few new instructions:

- make sure to include your version of `icuas22_competition` package (with your session.yml and any custom config files) to your team zip file. 
- once you detect the tag, publish the position of the tag in world coordinate frame as a `geometry_msgs/Point` message to topic `/red/tag_position_reconstructed`
- along with the position of the tag, you should publish annotated image of the tag as a proof of detection (annotation in form of a bounding box around the tag is sufficient). The image should be published as a `sensor_msgs/Image` to topic `/red/tag_image_annotated`
- make sure to release the ball by publishing gain on the uav topic `/red/uav_magnet/gain` and not on the ball magnet topic 

Also, make sure your solution can be built, as none of the submitted solutions could be built so no points can be awarded for this first run. 

Note that time limit and penalty points have not been set yet, as we will wait for a few successful runs to see how teams perform. Good luck to everyone!

**Comment by afaroo01** [comment]
Hi @fpetric, 
Is it necessary to do both or it is optional, to publish the annotated image along with the detected position of the artag? We had only published on the /red/tag_position_reconstructed topic the artag detected position and submitted our solution today. Will suffice?

**Reply by fpetric** [reply]
The evaluation script expects both, so you must do both.

**Reply by afaroo01** [reply]
Hi @fpetric @lmark1, we updated our last solution on on git had sent yesterday for evaluation. It has both the topics now added which when you build should be included. But we haven't received any reply for our evaluation nor it's receipt of confirmation. Do we expect evaluation to be done within a day, so if needed we can still make any changes before the end of May 8?

**Reply by fpetric** [reply]
Hi, 

check the latest anouncement #65. Sorry for not replying sooner.
